spec/models/refinery/jobs/job_spec.rb

```diff
@@ -13,9 +18,43 @@ module Jobs
           expect(FactoryGirl.build(:job, description: '')).not_to be_valid
 
           #Positive tests
-          expect(FactoryGirl.build(:job, {description: 1, title: 1})).to be_valid
+          expect(FactoryGirl.build(:job, description: 1, title: 1)).to be_valid
+        end
+      end
+
+      context 'update' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.